/**:
  ros__parameters:
    mode: "reactive"
    time_step: 0.1
    # traj_shape: "hover"
    # traj_shape: "hover_1.5_0_0"
    # traj_shape: "circle_r_1.0_sPerFullCircle_40"
    traj_shape: "circle_r_0.75_sPerFullCircle_40"
    traj_duration: 30
    actuator_failures: 
      # # Case 1
      # - act_ids: [3,1]
      #   intensity: 1.0
      #   start_time: 10
      # - act_ids: [4,1]
      #   intensity: 1.0
      #   start_time: 10
      ## Case 2
      # - act_ids: [1,1]
      #   intensity: 0.0
      #   start_time: 10
      # - act_ids: [1,2]
      #   intensity: 1.0
      #   start_time: 10
      ## Case 3
      - act_ids: [1,1]
        intensity: 0.0
        start_time: 20
      - act_ids: [2,1]
        intensity: 0.0
        start_time: 20
      - act_ids: [3,1]
        intensity: 0.0
        start_time: 20
    tuning:
      nominal:
        horizon: 20
        param_set: "P1"
        P1: 
          Q: [10.0, 10.0, 10.0, 1.0, 1.0, 0.1]
          R: [1.0, 1.0, 1.0]
          R_full: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
          P_mult: 10.0  # P = P_mult * Q
        P2: 
          Q: [5.0, 5.0, 10.0, 0.2, 0.2, 0.1]
          R: [1.0, 1.0, 1.0]
          R_full: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
          P_mult: 1.0  # P = P_mult * Q
      spiraling:
        horizon: 15
        param_set: "P1"
        # failsafe_mode: "hold" # "hold" or "continue_trajectory"
        failsafe_mode: "hold_center" # "hold" or "continue_trajectory"
        # failsafe_mode: "continue_trajectory" # "hold" or "continue_trajectory"
        P1:
          Q: [1, 1, 1, 1, 2]
          R: [0.1, 0.1, 0.01]
          P_mult: 1
          # Attention: The following parameters are only updated if the terminal set is recalculated!
          k_omega: 1.0
          max_acceleration: 0.0    
          time_scaling: 5
          empc_horizon: 10
      fb_lin:
        param_set: "P1"
        P1:
          Q: [5.0, 5.0, 0.2, 0.2, 100.0]
          R: [1.0, 1.0, 1.0]
          P_mult: 1.0
          Q_linfb: [0.1, 0.1, 1.0, 1.0, 1.0]
          R_linfb: [10.0, 10.0, 10.0] 
